test/api/buffer/api_test.go

```diff
@@ -88,6 +94,15 @@ func TestBufferApiE2E(t *testing.T) {
 			}
 		}
 
+		// Start worker node
+		clk := clock.NewMock()
+		clk.Set(time.Time{}.Add(time.Second))
+		opts := []dependencies.MockedOption{dependencies.WithClock(clk), dependencies.WithEtcdNamespace(etcdNamespace)}
+		workerDeps := bufferDependencies.NewMockedDeps(t, opts...)
+		workerDeps.DebugLogger().ConnectTo(testhelper.VerboseStdout())
+		_, err := service.New(workerDeps)
+		assert.NoError(t, err)
```

Comment:
@JakubMatejka toto som si nahodou vsimol, viem ze to nie je na review, ale radsej sa spytam.

Myslim, ze by si worker v API testoch nemusel potrebovat, aky je tu dovod?

Samotny task (create receiver/create export) bezi (by mal bezat) na API node nie?
Nemame tam ziadnu logiku typu:
- zaloz task a worker ho vykona
- mame tam iba zaloz a spusti task na aktualnej node (co IMHO staci)

Aj z tych logov co si vcera posielal to vyzeralo tak, ze to bezi na API node, preto som aj pisal, ze nerozumiem tomu "worker".

Reply:
Jo aha, tak to jsem nepochopil. Já myslel že ty tasky vykonává jen worker. 😅

Reply:
Ako mohol byt (ked sa ti to chce implementovat :slightly_smiling_face: ).
Ale jednoduchsie bude nechat to na API node.
Ono to len caka na vytvorenie danych resources a asi nenajdeme dovod, preco by to musel riesit worker.

Reply:
Pri vypinani API node sa na dane tasky pocka, to je implementovane v `task.Node`, cez `WaitGroup`.

Reply:
👍